manufacturers/Watteco/models/Vaqao/uplink_decoder.js

```diff
@@ -0,0 +1,41 @@
+let watteco = require("../../decode.js")
+let ngsild = require("../../ngsi-ld.js")
+
+let batch_param = [3, [{taglbl: 0,resol: 1, sampletype: 4,lblname: "occupancy", divide: 1, unit: ""},
+    { taglbl: 1, resol: 10, sampletype: 7,lblname: "temperature_1", divide: 100, unit: "CEL"},
+    { taglbl: 2, resol: 100, sampletype: 6,lblname: "humidity_1", divide: 100, unit: "P1"},
+    { taglbl: 3, resol: 10, sampletype: 6,lblname: "CO2", divide: 1, unit: "52"},
+    { taglbl: 4, resol: 10, sampletype: 6,lblname: "TVOC", divide: 1, unit: ""}]];
+
+let endpointCorresponder = {
+    concentration: ["TVOC", "CO2"],
+    temperature: ["temperature_1","temperature_2"],
+    humidity: ["humidity_1","humidity_2"],
+    pin_state:["violation_detection"]
+}
+
+function main() {
+    var payload = process.argv[3];
+    var time = process.argv[4];
+    // ********* Test pattern (uncomment to test behaviour) ********************
+        // Pattern batch
+        // payload = "424500000111008FC85EC128B4872088AD060442001E3322118A98782B";
```

Comment:
When trying with this payload, I get 
```
{"temperature_1":[{"type":"Property","value":26.3,"observedAt":"2024-06-13T07:41:12.976Z","unitCode":"CEL","datasetId":"urn:ngsi-ld:Dataset:Raw"},{"type":"Property","value":26.3,"observedAt":"2024-06-13T07:41:17.976Z","unitCode"
:"CEL","datasetId":"urn:ngsi-ld:Dataset:Raw"}],"humidity_1":[{"type":"Property","value":41,"observedAt":"2024-06-13T07:41:12.976Z","unitCode":"P1","datasetId":"urn:ngsi-ld:Dataset:Raw"},{"type":"Property","value":41,"observedAt"
:"2024-06-13T07:41:17.976Z","unitCode":"P1","datasetId":"urn:ngsi-ld:Dataset:Raw"}],"CO2":[{"type":"Property","value":1540,"observedAt":"2024-06-13T07:41:14.976Z","unitCode":"52","datasetId":"urn:ngsi-ld:Dataset:Raw"},{"type":"P
roperty","value":1500,"observedAt":"2024-06-13T07:41:19.976Z","unitCode":"52","datasetId":"urn:ngsi-ld:Dataset:Raw"}],"TVOC":[{"type":"Property","value":30,"observedAt":"2024-06-13T07:41:12.976Z","datasetId":"urn:ngsi-ld:Dataset
:Raw"},{"type":"Property","value":30,"observedAt":"2024-06-13T07:41:17.976Z","datasetId":"urn:ngsi-ld:Dataset:Raw"}]}
```
We can't have "temperature_1" in the entity, it needs to be "temperature"

Reply:
Should it ? Some of the Watteco sensors will return temperature_1 and temperature_2. How to handle it ? The 2 options I saw are 1. keeping this as a property name or 2. use of dataset.
I choose option 1 as mapping toward a temperature property would be made when connecting the Device to the observedEntity. 
Option 2 should be feasible as well.

Reply:
I don't really like option 1 (very basic example: if you do a query to get all Device that measures temperature, you won't get this one) but it is true that if we use 2 datasetId and try to map them to the same observedEntity, they will be mapped to the same datasetId on the observedEntity (urn:ngsi-ld:Dataset:Device:*devEui*:Raw). 
I still think we should go for option 2, we just need to do a simple evolution of the mapping module for it to work : 
if the source datasetId is urn:ngsi-ld:Dataset:*whatever*:Raw, the target datasetId would be urn:ngsi-ld:Dataset:Device:*devEui*:*whatever*:Raw (in this case *whatever* would be "Temperature1", "1" or "Probe1", smth like that)

Reply:
OK. 
For the time being I did not fund any mention of temperature_2 (or humidity_2) in the devices specifications (http://support.nke-watteco.com/wp-content/uploads/2017/02/LoRaWAN_Sensors_Application_Layer_Description.pdf) and codecs examples so they seem unused. So I simply changed temperature_1 to temperature